.github/actions/helm-default/action.yaml

```diff
@@ -43,6 +45,10 @@ runs:
           --helm-set=hubble.relay.image.tag=${{ inputs.image-tag }} \
           --helm-set=hubble.relay.image.useDigest=false"
 
+        if [ "${{ inputs.debug }}" == "true" ]; then
+          CILIUM_INSTALL_DEFAULTS+=" --helm-set=debug.enabled=true --helm-set=debug.verbose=envoy"
+        fi
+
         if [ -f "${{ inputs.chart-dir }}/../../../.github/actions/helm-default/ci-required-values.yaml" ]; then
           CILIUM_INSTALL_DEFAULTS+=" --values=${{ inputs.chart-dir }}/../../../.github/actions/helm-default/ci-required-values.yaml"
```

Comment:
I think this part should be guarded as well, as enabling cache mutation detection doesn't make sense for scale tests, given the memory implications.